src/main/java/uk/gov/hmcts/darts/annotation/service/impl/AnnotationServiceImpl.java

```diff
@@ -66,6 +79,46 @@ public Integer process(MultipartFile document, Annotation annotation) {
         return annotationEntity.getId();
     }
 
+    @Override
+    public AnnotationResponseDto downloadAnnotationDoc(Integer annotationId, Integer annotationDocumentId) {
```

Comment:
i don't see any checks that the user downloading the annotation has access to it, it shoudl be somethgin similar to Tonys:-
https://github.com/hmcts/darts-api/pull/1032/files#diff-3cb4a8770477be43d2a61bc42628126ac2d90df6ca290270f9d73f191a6c1ab2R25


or perhaps wait until his is merged and make use of it